.travis.yml

```diff
@@ -1,10 +1,11 @@
-language: elixir
-elixir:
-  - 1.3.2
-  - 1.4.0
-otp_release:
-  - 18.0
-  - 19.0
+matrix:
+  include:
+    - language: elixir
+      otp_release: 20.3
```

Comment:
Just to check, are we removing support for earlier than Elixir 1.6?

Reply:
Yes, redix dropped support for in 0.8 https://github.com/whatyouhide/redix/blob/master/CHANGELOG.md#v080

Reply:
👍OK, sounds good, I can approve PR.  Do you have a chance to resolve conflicts on this? I merged a cleanup PR and appears it introduced conflicts with this.

Reply:
sure, will fix the conflict